<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginService</class>
 <widget class="QWidget" name="LoginService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>339</width>
      <height>329</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelimage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labeluser">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelbirthday">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string>生日</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelsex">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string>个性签名:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGetClient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);</string>
     </property>
     <property name="text">
      <string>获取在线用户</string>
     </property>
    </widget>
    <widget class="QLabel" name="labeladdress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>479</width>
      <height>329</height>
     </rect>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>456</width>
       <height>177</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(138, 226, 52);
</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>481</width>
       <height>114</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>51</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(138, 226, 52);
</string>
          </property>
          <property name="text">
           <string>登录时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditStartTime">
          <property name="minimumSize">
           <size>
            <width>371</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>51</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(138, 226, 52);
font: 16pt &quot;Ubuntu&quot;;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>67</width>
            <height>51</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(138, 226, 52);
</string>
          </property>
          <property name="text">
           <string>当前时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditCurrent">
          <property name="minimumSize">
           <size>
            <width>371</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>51</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(138, 226, 52);
font: 16pt &quot;Ubuntu&quot;;
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>831</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEditMessage">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditSend">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelSendName">
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonsendall">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>群发信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRecv">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>接收信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
